chart/compass/values.yaml

```diff
@@ -229,6 +235,11 @@ global:
               give_up_after: 3s
               max_delay: 2000ms
 
+  tenantFetcher:
+    host: compass-tenant-fetcher.compass-system.svc.cluster.local
+    prefix: /tenants
```

Comment:
other prefixes are not extracted, extract all or inline all :)

Reply:
It's done - everything is extracted.